vars/packageBuildingPipelineDAOS.groovy

```diff
@@ -203,12 +203,81 @@ def call(Map pipeline_args) {
                             }
                         }
                     } //stage('Build on CentOS 7')
+                    stage('Build on CentOS 8') {
+                        when {
+                            beforeAgent true
+                            allOf {
+                                expression { distros.contains('centos8')}
```

Comment:
If you end up having to push another commit:
```suggestion
                                expression { distros.contains('centos8') }
```
to make the whitespace consistent.  And could you update the other instances of this particular inconsistency in this file while you are at it?

Reply:
Pushing updated commit with change.